devtools/platform-descriptor-json/src/main/filtered/quarkus.properties

```diff
@@ -8,10 +8,16 @@ kotlin-version=${kotlin.version}
 scala-version=${scala.version}
 scala-plugin-version=${scala-plugin.version}
 
-# Plugin metadata
-plugin-groupId=${project.groupId}
-plugin-artifactId=quarkus-maven-plugin
-plugin-version=${project.version}
```

Comment:
Removing plugin-version will break QuarkusJsonPlatformDescriptorResolver.getBundledPlatformQuarkusVersionOrNull()

Reply:
Is this targeting 1.8.x?

Reply:
yes it is targeting 1.8.x (should be alright right?). This is why I made a different PR for 1.7.x https://github.com/quarkusio/quarkus/pull/11439

Reply:
wait - why are we talking about removing plugin-version ?

Reply:
Yes, there was too much confusion in the code with it (it was used as quarkus-core-version), it is now split in 3:
```
quarkus-core-version=${project.version}
...
maven-plugin-version=${project.version}
...
gradle-plugin-version=${project.version}
```